src/components/views/dialogs/SeshatResetDialog.tsx

```diff
@@ -15,25 +15,24 @@ limitations under the License.
 */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import {_t} from "../../../languageHandler";
-import * as sdk from "../../../index";
 import {replaceableComponent} from "../../../utils/replaceableComponent";
 
+import BaseDialog from "./BaseDialog";
+import DialogButtons from "../elements/DialogButtons";
+
+interface IProps {
+    onFinished(): void;
+}
```

Comment:
Dialogs are a bit weird with their props, so this would become `interface IProps extends IDialogProps {}` 

Sorry, didn't clue in on this during the first round of review

Reply:
Fair in that case I could remove the interface altogether as `IDialogProps` is exactly what I'm looking for 🎉